src/vnsw/agent/vrouter/ksync/ksync_flow_memory.cc

```diff
@@ -406,12 +423,38 @@ void KSyncFlowMemory::MapSharedMemory() {
             << "> : " << strerror(errno));
         assert(0);
     }
+#else
+    flow_table_ = GetFlowTableMemoryFromWindowsPipe();
```

Comment:
This can be removed, because that code will be called only on FreeBSD.

Reply:
This is correct.